.github/workflows/meeting-notes.yml

```
@@ -37,29 +37,30 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: '3.10'
-      - name: Install dateutil
-        if: github.event_name == 'schedule'
-        run: python -m pip install python-dateutil
       - name: Check if we need to run this week
         if: github.event_name == 'schedule'
         id: check-wednesday
         shell: python
         run: |
-          import os
-          from datetime import datetime
-          from dateutil.rrule import rrule, WEEKLY, WE
-          today = datetime.today()
-          print("Today is", today)
-          every_two_wednesdays = rrule(WEEKLY, interval=2, dtstart=datetime(2023, 3, 15), wkst=WE)
-          for wednesday in every_two_wednesdays:
-            print("Testing if today matches...", wednesday)
-            if wednesday.date() == today.date():
-              print("Match! Job should run.")
+          import datetime, sys
+          year, week, weekday = dt.isocalendar()
```

```suggestion
          year, week, weekday = datetime.today().isocalendar()
```

You mean this, right?